pom.xml

```diff
@@ -41,6 +41,7 @@
 		<ugandaemrVersion>3.0.1</ugandaemrVersion>
 		<ugandaemrPocVersion>1.0.4-SNAPSHOT</ugandaemrPocVersion>
```

Comment:
@reagan-meant increase UgandaEMR POC to 1.0.5-SNAPSHOT which is the current version

Reply:
Done